src/go/wsl-helper/pkg/wsl-utils/version_test.go

```diff
@@ -0,0 +1,113 @@
+//go:build windows
+// +build windows
+
+package wslutils
```

Comment:
Can we separate the test namespace from the actual code? e.g. `wslutils_test`

Reply:
That would require exporting `getPackageNames`, etc. which seems like a bad abstraction (the callers shouldn't need to know about those)…